05_testing_and_ci/examples/python_testing/test_operations_unittests.py

```diff
@@ -0,0 +1,80 @@
+"""
+Tests for mathematical operations functions.
+"""
+from operations import find_max, find_mean
+from unittest import TestCase
+import csv
+
+
+class TestOperations(TestCase):
+    """
+    Test suite for mathematical operations functions.
+    """
+    def setUp(self):
+        # Fixture
+        self.data = [43, 32, 167, 18, 1, 209]
+        self.data2 = [3, 13, 33, 23, 498]
+
+    # Unit test
+    def test_find_max(self):
+        """
+        Test operations.find_max 
+        """
+        # Expected result
+        expected_max = 209
```

Comment:
Is it good practice that the expected outcome is not part of the fixture? (I don't know, just asking)

Reply:
There is no set rule as to what exactly is included in the fixture, but the general idea is that the fixture is really input data towards testing the intended function. I would keep them separate for clarity.